modules/03 Front End Development/Assignments/54/index5.html

```diff
@@ -0,0 +1,207 @@
+                location.reload();
+            }
+            if (document.getElementById('6').innerText === '0' && document.getElementById('4').innerText === '0' && document.getElementById('5').innerText === '0') {
+                document.body.innerHTML = "";
+                document.body.style.opacity = 0;
+                setTimeout(() => alert(name0.value + ' win! click "OK" to restart'), 0);
+                location.reload();
+            }
+            if (document.getElementById('7').innerText === '0' && document.getElementById('8').innerText === '0' && document.getElementById('9').innerText === '0') {
+                document.body.innerHTML = "";
+                document.body.style.opacity = 0;
+                setTimeout(() => alert(name0.value + ' win! click "OK" to restart'), 0);
+                location.reload();
+            }
+            if (document.getElementById('1').innerText === '0' && document.getElementById('4').innerText === '0' && document.getElementById('7').innerText === '0') {
+                document.body.innerHTML = "";
+                document.body.style.opacity = 0;
+                setTimeout(() => alert(name0.value + ' win! click "OK" to restart'), 0);
+                location.reload();
+            }
+            if (document.getElementById('2').innerText === '0' && document.getElementById('5').innerText === '0' && document.getElementById('8').innerText === '0') {
+                document.body.innerHTML = "";
+                document.body.style.opacity = 0;
+                setTimeout(() => alert(name0.value + ' win! click "OK" to restart'), 0);
+                location.reload();
+            }
+            if (document.getElementById('3').innerText === '0' && document.getElementById('6').innerText === '0' && document.getElementById('9').innerText === '0') {
+                document.body.innerHTML = "";
+                document.body.style.opacity = 0;
+                setTimeout(() => alert(name0.value + ' win! click "OK" to restart'), 0);
+                location.reload();
+            }
+            if (document.getElementById('1').innerText === '0' && document.getElementById('5').innerText === '0' && document.getElementById('9').innerText === '0') {
+                document.body.innerHTML = "";
+                document.body.style.opacity = 0;
+                setTimeout(() => alert(name0.value + ' win! click "OK" to restart'), 0);
+                location.reload();
+            }
+            if (document.getElementById('3').innerText === '0' && document.getElementById('5').innerText === '0' && document.getElementById('7').innerText === '0') {
+                document.body.innerHTML = "";
+                document.body.style.opacity = 0;
+                setTimeout(() => alert(name0.value + ' win! click "OK" to restart'), 0);
+                location.reload();
+            }
+        }
+    </script>
+</body>
+
+</html>
```

Comment:
correct, but you missed: 'when "Game Over" appears, hide the rest of the elements and display only the text and a button for starting a new game'